modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,157 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService ingress controller
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller  overrides this, allowing for manual control.
+
+[NOTE]
+====
+This feature and procedure applies only to AWS. 
+====
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your IngressController.
+
+.Procedure
+To create an Ingress Controller with manually specified load balancer subnets, you can follow these steps:
+
+. Create a custom resource (CR) file.
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name>
+spec:
+  domain: <domain>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+  dnsManagementPolicy: Managed
+----
+
+. Create a custom resource (CR) file.
+
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name> <1>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic
+          classicLoadBalancer: <3>
+            subnets:
+              ids: <4>
+              - <subnet> <5>
+              - <subnet>
+              - <subnet>
+----
+<1> Replace `<name>` with a name for the Ingress Controller
+<2> Replace `<domain>` with the DNS name serviced by the Ingress Controller
+<3> You can also use the `networkLoadBalancer` field if using an NLB
+<4> You can optionally specify subnet by name using the `names` field
```

Comment:
```suggestion
<2> Replace `<domain>` with the DNS name serviced by the `IngressController`.
<3> You can also use the `networkLoadBalancer` field if using an NLB.
<4> You can optionally specify a subnet by name using the `names` field.
```

Reply:
For #4, I am wondering, is this option _instead_ of specifying the subnet by ID? If so, it may be clarifying to say that.